charts/console/values.yaml

```diff
@@ -6,7 +6,7 @@ replicaCount: 1
 
 image:
   registry: docker.redpanda.com
-  repository: vectorized/console
+  repository: redpandadata/console
```

Comment:
Let's revert this change in this PR and we'll do it in the bump PR for 23.1.1. This would be a breaking change as anybody who uses the chart in an air-gapped environment would not have an image with this repository reference in their cache, causing their pod to be unable to be started.

Since (apparently) all Y.Q Redpanda versions are considered major version changes, we'll make this change there and bump the major version of the chart.